butane_core/src/fkey.rs

```diff
@@ -23,8 +23,7 @@ use fake::{Dummy, Fake, Faker};
 ///   blog: ForeignKey<Blog>,
 ///   ...
 /// }
-///
-#[derive(Default)]
+#[derive(Clone, Default, Deserialize, Serialize)]
```

Comment:
Does this correctly dereferences the ForeignKey struct into the inner type when serialising?

Reply:
yup

Reply:
Awesome, thought you'd have to do a manual impl.